webclient/heslar/hesla.py

```diff
@@ -13,6 +13,9 @@
 SPECIFIKACE_DATA_PRESNE: Final = 885  # HES-000887
 TYP_DJ_SONDA_ID: Final = 1070  # HES-001072
 TYP_DJ_KATASTR: Final = 1071 # HES-001073
+TYP_DJ_CELEK: Final = 1068 # HES-001073
+TYP_DJ_CAST: Final = 1069 # HES-001073
+TYP_DJ_LOKALITA: Final = 1072 # HES-001073
```

Comment:
@jnihnat Tady platí (ID se po nové migraci změní, ident_cely zůstane):
1068 # HES-001070
1069 # HES-001071
1072 # HES-001074

Reply:
tohle ale predpokladam bude treba resit pres celou aplikaci, vsechny hesla v hesla.py jsou na ID a budou se muset predelat na identy nasledne i v aplikaci.

Reply:
Ano, máme na to issue (#749). Jen si k tomu rovnou chci uvádět správné komentáře, ať pak není zmatek. Jak tam bylo třikrát totéž, bylo by to vyloženě matoucí.

Reply:
Honzo, díky za úpravu.